DependencyInjection/Compiler/MiddlewaresPass.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace Doctrine\Bundle\DoctrineBundle\DependencyInjection\Compiler;
+
+use Doctrine\Bundle\DoctrineBundle\Middleware\ConnectionNameAwareInterface;
+use Symfony\Component\DependencyInjection\ChildDefinition;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+use function array_keys;
+use function is_subclass_of;
+use function sprintf;
+
+class MiddlewaresPass implements CompilerPassInterface
+{
+    /** @var string  */
+    private $connexionDefsParam;
```

Comment:
```suggestion
    private $connectionDefsParam;
```
:fr: spotted :slightly_smiling_face:

Reply:
😅 thank you